monorepo-folders/packages/temporal-workflows/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "temporal-workflows",
```

Comment:
This also includes activities maybe rename or restructure?

Reply:
i was a bit uncomfortable with it too but couldnt think of a better name

customer definitely wanted to organize activities that are only used by a certain workflow inside the folder for that workflow, even if it is an arbitrary organization.

perhaps `temporal-src`?